torchgeo/datasets/sen12ms.py

```diff
@@ -247,6 +334,63 @@ def _check_integrity(self) -> bool:
                 return False
         return True
 
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample return by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional suptitle to use for figure
+
+        Returns;
+            a matplotlib Figure with the rendered sample
+
+        .. versionadded:: 0.2
+        """
+        rgb_indices = []
+        for band in self.RGB_BANDS:
+            if band in self.bands:
+                rgb_indices.append(self.bands.index(band))
+            else:
+                raise ValueError("Dataset doesn't contain some of the RGB bands")
+        import pdb
+
+        pdb.set_trace()
+        image, _ = sample["image"][rgb_indices, ...], sample["mask"]
+        ncols = 2
+
+        showing_predictions = "prediction" in sample
+        if showing_predictions:
+            _ = sample["prediction"]
```

Comment:
This line doesn't do anything

Reply:
I haven't actually plotted the mask or the prediction because of my question in the PR request about how to handle the 4 channel MODIS land cover mask, this is just to have a skeleton.

Reply:
Thanks, marked this as a draft since it's still a work-in-progress